src/OrchardCore/OrchardCore.Feeds.Abstractions/Models/FeedMetadata.cs

```diff
@@ -1,10 +1,12 @@
-﻿namespace OrchardCore.Feeds.Models
+namespace OrchardCore.Feeds.Models
 {
     /// <summary>
     /// Used to gather custom Feed properties for a content item.
     /// </summary>
     public class FeedMetadata
     {
+        public bool EnableFeedProxyUrl { get; set; } = true;
```

Comment:
The name here is wrong, the property should be named `DisableRssFeed` such that we "check" the settings to change the default (which is to display the feed).

We need to remove the work `proxy` here because it has nothing to do with proxies.

Reply:
This change the intention to disable the feed by default

Reply:
No, `EnableXYZ { get; set; } = true;` is the same as `DisableXYZ { get; set; } = false;` in terms of defauts